src/System.Data.Common/ref/System.Data.Common.cs

```diff
@@ -56,7 +56,7 @@ public abstract partial class Constraint
         protected virtual System.Data.DataSet _DataSet { get { throw null; } }
         [System.ComponentModel.DefaultValueAttribute("")]
         public virtual string ConstraintName { get { throw null; } set { } }
-        [System.ComponentModel.Browsable(false)]
+        [System.ComponentModel.BrowsableAttribute(false)]
```

Comment:
Is this change from `Browsable` to `BrowsableAttribute` to align how the other attributes are being used in the file?